bin/seeds.js

```diff
@@ -0,0 +1,103 @@
+      showtimes: ['13:40', '15:50', '19:00', '21:20', '23:50']
+    },
+    {
+      title: 'Thoroughbreds',
+      director: 'Cory Finley',
+      stars: ['Anya Taylor-Joy', 'Olivia Cooke', 'Anton Yelchin'],
+      image:
+        'https://images-na.ssl-images-amazon.com/images/M/MV5BNDcyNDA4NDAzN15BMl5BanBnXkFtZTgwODQxMDQ5NDM@._V1_UX182_CR0,0,182,268_AL_.jpg',
+      description:
+        'Two upper-class teenage girls in suburban Connecticut rekindle their unlikely friendship after years of growing apart. Together, they hatch a plan to solve both of their problems-no matter what the cost.',
+      showtimes: ['13:00', '15:30', '18:00', '20:10', '22:40']
+    },
+    {
+      title: 'The Leisure Seeker',
+      director: 'Paolo Virzì',
+      stars: ['Helen Mirren', 'Donald Sutherland', 'Janel Moloney'],
+      image:
+        'https://images-na.ssl-images-amazon.com/images/M/MV5BMTg1NTg2MzcyNF5BMl5BanBnXkFtZTgwNjMwMDIzNDM@._V1_UX182_CR0,0,182,268_AL_.jpg',
+      description:
+        'A runaway couple goes on an unforgettable journey in the faithful old RV they call The Leisure Seeker, traveling from Boston to The Ernest Hemingway Home in Key West. They recapture their passion for life and their love for each other on a road trip that provides revelation and surprise right up to the very end.',
+      showtimes: ['13:40', '15:50', '19:00', '21:20', '23:50']
+    },
+    {
+      title: 'Black Panther',
+      director: 'Ryan Coogler',
+      stars: ['Chadwick Boseman', 'Michael B. Jordan', "Lupita Nyong'o"],
+      image:
+        'https://images-na.ssl-images-amazon.com/images/M/MV5BMTg1MTY2MjYzNV5BMl5BanBnXkFtZTgwMTc4NTMwNDI@._V1_UX182_CR0,0,182,268_AL_.jpg',
+      description:
+        "T'Challa, the King of Wakanda, rises to the throne in the isolated, technologically advanced African nation, but his claim is challenged by a vengeful outsider who was a childhood victim of T'Challa's father's mistake.",
+      showtimes: ['13:50', '16:20', '19:20', '22:10']
+    },
+    {
+      title: 'Red Sparrow',
+      director: 'Francis Lawrence',
+      stars: ['Jennifer Lawrence', 'Joel Edgerton', 'Matthias Schoenaerts'],
+      image:
+        'https://images-na.ssl-images-amazon.com/images/M/MV5BMTA3MDkxOTc4NDdeQTJeQWpwZ15BbWU4MDAxNzgyNTQz._V1_UX182_CR0,0,182,268_AL_.jpg',
+      description:
+        "Ballerina Dominika Egorova is recruited to 'Sparrow School,' a Russian intelligence service where she is forced to use her body as a weapon. Her first mission, targeting a C.I.A. agent, threatens to unravel the security of both nations.",
+      showtimes: ['13:00', '15:30', '18:00', '20:10', '22:40']
+    }
+  ];
+  
+
+
+mongoose
+  .connect('mongodb://localhost/express-cinema', {useNewUrlParser: true})
```

Comment:
Get used to add also the connection string among the variables in the .env file. In a real app this will be the connection string to a remote database

Reply:
Ok, good to know! What is the best way to do this? Do you have an example of what to write in the .env file?  Thx!

Reply:
In your .env file you can write _MONGODB_URI=mongodb://localhost/express-cinema_ and then change the connection to _mongoose
  .connect(process.env.MONGODB_URI)_